source/libs/utils.ts

```diff
@@ -86,3 +86,7 @@ export function getOP(): string {
 
 	return select('.timeline-comment-header-text .author')!.textContent!;
 }
+
+export function idx(target: any, path: string[]): any | null | undefined {
```

Comment:
`idx` is not a good name

Use `unknown` instead of `any` whenever possible.
```suggestion
export function idx(target: unknown, path: string[]): unknown {
```

Reply:
> `idx` is not a good name

Apparently that's what [Facebook uses](https://github.com/facebook/Docusaurus/pull/917#discussion_r213177032), every library that does this has a weird name anyway.

Reply:
Adding `AnyObject` as type to `target` gives type-not-found error while running ava, it looks like types from `globals.d.ts` are not included while running tests.

But fixed that using `ts-node/register/transpile-only` option in `package.json`, which makes the tests run way faster too. Maybe worth looking into adding this!?

Reply:
Transpile-only doesn’t actually check the types, so it defeats the purpose of using typescript